pkg/cluster/registry/factory/factory.go

```diff
@@ -10,36 +10,35 @@ import (
 
 var (
 	// Fty is the global registry factory
-	Fty = NewFactory()
+	Fty = NewRegistryCache()
 )
 
 // nolint
 //
 //go:generate mockgen -source=$GOFILE -destination=../../../../mock/pkg/cluster/registry/factory/factory_mock.go -package=mock_factory
```

Comment:
Could you run "go generate ./..." to re-generate mock files?

Reply:
ok